modular_skyrat/modules/loadouts/loadout_items/loadout_datum_pocket.dm

```diff
@@ -172,7 +172,42 @@ GLOBAL_LIST_INIT(loadout_pocket_items, generate_loadout_items(/datum/loadout_ite
 /datum/loadout_item/pocket_items/cross
 	name = "Ornate Cross"
 	item_path = /obj/item/crucifix
-	restricted_roles = list(JOB_CHAPLAIN)
```

Comment:
(i thought this item was kinda stupid to gatelock to a singular role and kinda removed this part for the religion arpee; didn't think it's worth an entire changelog part but i mean if you wanna)

Reply:
> You SHOULD put it in the Changelog so that players know they can equip it now.

real

Reply:
It's got a force of 10 and a throwforce of 15, that could probably be toned down a smidge.

Reply:
> It's got a force of 10 and a throwforce of 15, that could probably be toned down a smidge.

Done.